subsys/net/lib/shell/iface.c

```diff
@@ -817,6 +818,73 @@ static int cmd_net_default_iface(const struct shell *sh, size_t argc, char *argv
 	return 0;
 }
 
+static int cmd_net_link_speed(const struct shell *sh, size_t argc, char *argv[])
```

Comment:
maybe this function should only be available when `CONFIG_ETH_PHY_DRIVER` is defined